msteams-platform/tabs/how-to/access-teams-context.md

```diff
@@ -48,7 +48,8 @@ Use placeholders in your configuration or content URLs. Microsoft Teams replaces
 * [{app.locale}](/javascript/api/@microsoft/teams-js/app.appinfo#@microsoft-teams-js-app-appinfo-locale): The current locale of the user formatted as *languageId-countryId*, for example `en-us`. (Known as `{locale}` before TeamsJS v.2.0.0).
 
 > [!NOTE]
-> The previous `{upn}` placeholder is now deprecated. For backward compatibility, it is currently a synonym for `{user.loginHint}`.
+> - The previous `{upn}` placeholder is now deprecated. For backward compatibility, it is currently a synonym for `{user.loginHint}`.
+> - Mobile (Android and iOS) versions of Microsoft Teams currently support only TeamsJS v.1.x placeholders.
```

Comment:
>v.1 [](http://example.com/codeflow?start=88&length=3)

Should there be a period between "v" and "1" ? I naturally don't know what our documentation standard is on this but all other references on this page to versions don't have a period here.

Actually they also refer to v1 as "v1" and not "v.1.x", in case we want to be consistent.

Reply:
Good point. Updated for consistency--